.gitignore

```diff
@@ -3,3 +3,4 @@
 .build
 *.swp
 release-tools
+.idea
```

Comment:
?

Reply:
`.idea` is the folder created by JetBrains IDEs to store project metadata/configuration